napari/layers/_labels_layer/model.py

```diff
@@ -187,7 +187,7 @@ def brush_size(self):
     @brush_size.setter
     def brush_size(self, brush_size):
         self._brush_size = int(brush_size)
-        self.cursor_size = self._brush_size/self._get_rescale()
+        self.cursor_size = self._brush_size/self.scale_factor
```

Comment:
The changes to this file make me wonder whether Labels should be a subclass of Image, rather than a sister implementation. Thoughts? (This can obviously wait for a later PR, it's just a good time to have the discussion I think.)

Reply:
yes - it might make a lot of sense to have Labels as a subclass of Image. I think they have a lot in common and that you'd just need to overwrite a few properties / methods, customize a few others to make it work. I'm also thinking about how we'd extend image pyramid support to labels too, and making labels be a subclass and just adding image pyramid support to the main image layer would be one way.

Reply:
Ok, we'll leave that to another PR, but glad we are in agreement. =)

Reply:
we could even think about making a common class for all our mesh layers (i.e. shapes, markers, vectors) but i don't think we're ready for that yet